content/tutorials/spatial_standards_vector/index.Rmd

```diff
@@ -0,0 +1,202 @@
+---
+title: "How to use open vector file formats in R: GeoPackage & GeoJSON"
+description: "A simple tutorial to demonstrate the use of *.gpkg and *.geojson files in R"
+author: "Floris Vanderhaeghe"
+date: 2019-11-19
+categories: ["r", "gis"]
+tags: ["gis", "r"]
+output: 
+    md_document:
+        preserve_yaml: true
+        variant: gfm
+---
+
+```{r include = FALSE}
+library(knitr)
+opts_chunk$set(
+  echo = TRUE
+)
+unlink(c("sac.*", "habitatstreams.*"))
+```
+
+
+This tutorial uses a few basic functions from the [dplyr](https://dplyr.tidyverse.org) and [sf](https://r-spatial.github.io/sf/) packages.
+While only a few functions are used, you can use the previous hyperlinks to access the tutorials (vignettes) of these packages for more functions and information.
+
+```{r warning=FALSE, message=FALSE}
+options(stringsAsFactors = FALSE)
+library(tidyverse)
+library(sf)
+library(inborutils)
+```
+
+You find a bit more background about 'why and what', regarding the below open standards, in [a separate post](../../articles/geospatial_standards/) on this website.
+
+In short, the GeoPackage and GeoJSON formats are ideal for exchange, publication, interoperability & durability and to open science in general.
+
+## How to make and use GeoPackages (`*.gpkg`)
+
+### Making a GeoPackage from a geospatial `sf` object in R
```

Comment:
I suggest to move this section to the end.

Reply:
not done, see earlier remark (geojson has more complexities / limitations for the user)